editor/editor_node.cpp

```diff
@@ -2044,7 +2044,9 @@ void EditorNode::_dialog_action(String p_file) {
 
 		case FILE_EXPORT_MESH_LIBRARY: {
 			Ref<MeshLibrary> ml;
-			if (file_export_lib_merge->is_pressed() && FileAccess::exists(p_file)) {
+			const Dictionary &fd_options = file_export_lib->get_selected_options();
+			bool file_export_lib_merge = fd_options.has(TTR("Merge With Existing")) ? (bool)fd_options[TTR("Merge With Existing")] : true;
```

Comment:
```suggestion
			bool file_export_lib_merge = (bool)fd_options.get(TTR("Merge With Existing"), true);
```

Reply:
tbh accessing values by their translated names is somewhat awkward. I think they should be subject to `auto_translate` property and the dialogs could automatically send translated options to the native dialog call.

Although the DisplayServer callbacks return the final option name, so I'd require some internal magic to keep track of the original strings. Maybe it's not worth it then, idk. It's something that can be improved later.